redux/js/actions/index.js

```diff
@@ -0,0 +1,47 @@
+import shop from '../../../common/api/shop';
```

Comment:
https://github.com/joeybaker/remapify might be a nice improvement for common things to remove the relative path